var.tfvars

```diff
@@ -47,7 +47,8 @@ cluster_id                  = ""         # It will use random generated id with
 
 #network_type               = "SRIOV"
 #scg_id                     = "df21cec9-c244-4d3d-b927-df1518672e87"
-
+#sriov_vnic_failover_adapter = 2
```

Comment:
Better keep the default value of `1` here.

Reply:
Good point. Will update the var.tfvars accordingly.